FUNDERS.yaml

```diff
@@ -238,3 +238,17 @@ eurosciencegateway:
     funding_statement: |
         EuroScienceGateway was funded by the European Union programme Horizon Europe (HORIZON-INFRA-2021-EOSC-01-04) under grant agreement number 101057388 and by UK Research and Innovation (UKRI) under the UK government’s Horizon Europe funding guarantee grant number 10038963.
 
+fnso2019:
+    name: "OpenMetaPaper"
+    short_name: "FNSO"
+    joined: 2023-05
+    avatar: "https://gallantries.github.io/assets/images/fnso.png"
+    github: false
+    funder: true
+    url: "https://www.ouvrirlascience.fr/national-fund-for-open-science/"
+    funder_name: National Fund for Open Science
+    funding_id: AAPFNSO2019OpenMetaPaper-14026
+    funding_statement: |
+        This project ([`AAPFNSO2019OpenMetaPaper-14026`](https://www.ouvrirlascience.fr/submit-your-data-its-published/)) is funded with the support of the national Fund for Open Science programme of the French Minister of Higher Education, Research and Innovation. Their funding has supported development of data/metadata management tools and related tutorial for Ecology and FAIR topics.
+        ![French research ministry flag with the text: with the support of the National Fund for Open Science programme of french Minister of Higher Education, Research and Innovation](https://lamy-environnement.com/wp-content/uploads/2023/03/Logo-MESR.png)
```

Comment:
the alt text :heart_eyes: :heart_eyes: merci !

Reply:
not sure if you are saying it is very good or bad ? ;) sorry

Reply:
yes it's good! and always so nice when people write it and I do not have to nag them :)